src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -166,12 +166,12 @@ func (d *Dumper) DumpCluster() error {
 				continue
 			}
 			location = filepath.Join(d.location, ns.Name, pod.Name, "/pt-summary.txt")
-			component := d.crType
-			if d.crType == "psmdb" {
+			component := resourceType(d.crType)
+			if component == "psmdb" {
 				component = "mongod"
 			}
 			if pod.Labels["app.kubernetes.io/component"] == component {
-				output, err = d.getPodSummary(d.crType, pod.Name, pod.Labels["app.kubernetes.io/instance"], tw)
+				output, err = d.getPodSummary(resourceType(d.crType), pod.Name, pod.Labels["app.kubernetes.io/instance"], tw)
```

Comment:
feel free to ignore
```suggestion
				output, err = d.getPodSummary(component, pod.Name, pod.Labels["app.kubernetes.io/instance"], tw)
```

Reply:
pt-mongodb-summary wouldn't run in this case. `getPodSummary` method expects the first argument to be either `pxc` or `psmdb`. While running against PSMDB, the `component` variable will have `mongod` value

Reply:
in such a case why do we expect `psmdb` value in line 170 from the `component` variable?